amy/workshops/models.py

```diff
@@ -140,6 +140,10 @@ class Meta:
 
 class MembershipManager(models.Manager):
     def annotate_with_seat_usage(self):
+        cldt_tag = Tag.objects.get(name="CLDT")
+        # TTT/ITT included
+        ttt_tags = Tag.objects.exclude(name__in=["SWC", "DC", "LC", "WiSE", "CLDT"])
```

Comment:
In here (L143, L145) use tag names:

```python
cldt_tag_name = "CLDT"
ttt_tag_names = ["TTT", "ITT"]
```

Then the lookups using `ttt_tags` or `cldt_tag` have to be changed:
1. from `task__event__tags=[ttt_tags]` to `task__event__tags__name__in=ttt_tag_names`
2. from `task__event__tags=[cldt_tag]` to `task__event__tags__name__in=[cldt_tag_name]`